gums.config.template

```diff
@@ -86,6 +86,16 @@
 			sslCAFiles='/etc/grid-security/certificates/*.0'/>
 
 		<vomsServer
+			name='cigi2'
```

Comment:
Please remove the original `cigi` server entry and name this one `cigi` instead of `cigi2`.